core/platform/bulk_email/mailchimp_bulk_email_services.py

```diff
@@ -253,14 +253,14 @@ def add_or_update_user_status(
             'status': 'subscribed',
             'tags': [tag],
             'merge_fields': {
-                'NAME': merge_fields['NAME']
+                'NAME': merge_fields.get('NAME', None)
```

Comment:
I think, if no name is provided, then just don't send this field? Ditto below.

Reply:
Done, Thanks